cordapp-cpk2/src/test/kotlin/net/corda/plugins/cpk2/WithDependentCordappTest.kt

```diff
@@ -114,16 +112,16 @@ class WithDependentCordappTest {
         val cpk = artifacts.single { it.toString().endsWith(".cpk") }
         assertThat(cpk).isRegularFile
 
-        val cordappDepsFile = testProject.buildDir.resolve("DependencyConstraints")
-        assertThat(cordappDepsFile).isRegularFile
-        val cordappDependencyConstraints = cordappDepsFile.toFile().inputStream()
-            .use(::loadDependencyConstraints)
-        assertThat(cordappDependencyConstraints)
-            .noneMatch { it.fileName == "biz.aQute.bnd.annotation-${bndVersion}.jar" }
-            .noneMatch { it.fileName == "slf4j-api-${librarySlf4jVersion}.jar" }
-            .noneMatch { it.fileName == "slf4j-api-${cordaSlf4jVersion}.jar" }
-            .anyMatch { it.fileName == "guava-${libraryGuavaVersion}.jar" }
-            .anyMatch { it.fileName == "commons-io-$commonsIoVersion.jar" }
-            .anyMatch { it.fileName == "library.jar" }
+        val contractCpk = testProjectDir.resolve("cordapp")
+            .resolve("build")
+            .resolve("libs")
+            .resolve("cordapp-cordapp.cpk")
```

Comment:
I think this should be (after tweaks to `with-dependent-cordapp` script):
```kotlin
val contractCpk = testProject.buildDir.resolve("cordapp-cordapp.cpk")
assertThat(contractCpk).isRegularFile
assertThat(listLibrariesForCpk(contractCpk))
   // etc...
```
I prefer the test projects to make their "interesting" artifacts readily available, rather than have the tests rummage through an arbitrary directory structure.

Reply:
I agree, that's much better. Thank you for your help.